client.go

```diff
@@ -19,12 +19,13 @@ type Client struct {
 	username   string
 	password   string
 	httpClient *http.Client
+	useBetaApi bool
 }
 
 // NewClient returns a new client
 // with the given credential
 // for the given testrail domain
-func NewClient(url, username, password string) (c *Client) {
+func NewClient(url, username, password string, useBetaApi ...bool) (c *Client) {
```

Comment:
Why using the `...` here?

Reply:
That's the way I found to make useBetaApi parameter as optional, so people consuming this function won't be affected with the change. I saw the same approach in other functions that have optional parameters. e.g. https://github.com/renzors/testrail/blob/3ab3c62b1fdc7aa566e568972586b6cbb7e89569/milestone.go#L34